### PR TITLE
Add docker configs to default paths of files and HTTP(S) ports

### PR DESCRIPTION
**Pull request by lprat**
Hi,

I changed docker-compose.yaml and config.env to permit simply change:
- local path different of default config (nginx conf, timeksetch conf, pg db, elsatic db)
- port bind on docker host (change default bind 80/443)
without modify docker-compose.yaml .

I added healtcheck in docker-compose to monit simply.

Thanks.
Lionel

### CODE REVIEW

**Comment by berggren** [comment]
Thanks @lprat ! I'll take a look at this as soon as time permits.

**Comment by berggren** [comment]
@lprat I'm picking this up again. Since this was filed we have migrated to OpenSearch. Can you fix the conflicts, and change the var names. 

Let me know, and we get this merged.

**Comment by lprat** [comment]
I added "networks" to isolate the containers from which it is not necessary to have outside/inside traffic

File: docker/release/docker-compose.yml [hunk 1]
```diff
@@ -51,21 +69,49 @@ services:
...
-    volumes:
-      - ./data/postgresql:/var/lib/postgresql/data
+    healthcheck:
+      test: ["CMD-SHELL", "bash -c '</dev/tcp/127.0.0.1/5432' || exit 1"]
+      interval: 30s
+      timeout: 5s
+      retries: 5
+    networks:
+      - timesketch_internal
+    volumes:https://github.com/cybersante
```
Comment (jaegeral) [inline docker/release/docker-compose.yml:79]: Not sure what that github repo here means, could you clarify @lprat

Comment (lprat) [reply docker/release/docker-compose.yml]: Sorry it's mistake! I fixed

**Review by berggren — COMMENTED** [review]
Simplified this PR a bit, removed the health checks for now and renamed the variables.

**Review by berggren — APPROVED** [review]

**Pull request merged**